api/provider/aws/service_update.go

```diff
@@ -0,0 +1,70 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func (s *ServiceProvider) Update(req models.UpdateServiceRequest) error {
+	serviceID := req.ServiceID
+	deployID := req.DeployID
+	fqServiceID := addLayer0Prefix(s.Config.Instance(), serviceID)
+
+	serviceTags, err := s.TagStore.SelectByTypeAndID("service", serviceID)
```

Comment:
I think all of this logic should go to a helper function in `common.go`:

```
func lookupEntityEnvironmentID(store tag.Store, entityType, entityID string) (string, error) { ... }
```

this would be used by services, tasks, and loadbalancers

Reply:
✅